app/lib/lets-encrypt.server.ts

```diff
@@ -160,7 +188,16 @@ class LetsEncrypt {
       // For testing and local development, let's use an ad-hoc generated key
 
       this.#accountKey = (await acme.crypto.createPrivateKey()).toString();
-      this.#directoryUrl = acme.directory.letsencrypt.production;
+      this.#directoryUrl = 'https://127.0.0.1:14000/dir';
```

Comment:
We'll have to make this configurable in a follow-up (I know you know this, just documenting).

Reply:
Sure, we can make an ENV for example